vendor/gems/ruby_spec_helpers/lib/ruby_spec_helpers/capybara_configuration.rb

```diff
@@ -33,11 +33,7 @@
   config.javascript_driver = driver
 
   capybara_wait_time = ENV.fetch('CAPYBARA_WAIT_TIME', 10).to_i
-  if config.respond_to? :default_max_wait_time=
-    config.default_max_wait_time = capybara_wait_time
-  else
-    config.default_wait_time = capybara_wait_time
-  end
+  config.default_max_wait_time = capybara_wait_time
```

Comment:
Capybara.config is deprecated and latest capybara has a bug in its deprecation warnings that don't allow you to call `respond_to?` anymore so I'm getting ahead of ruby_spec_helpers here.